__tests__/app.test.js

```diff
@@ -26,6 +26,17 @@ describe("/api/topics", () => {
   });
 });
 
+describe("/api", () => {
+  it("GET - 200: Should return a JSON describing all the available endpoints.", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.endpoints).toEqual(expect.any(String));
```

Comment:
This is a brittle test. Is there a way of checking that the information received is exactly the same as what is in your endpoints.json?